src/tests/test_error.rs

```diff
@@ -0,0 +1,50 @@
+use crate::error::Error;
+use cosmwasm_std::{WasmMsg, WasmQuery};
+
+#[test]
+fn instantiating_error_should_work() {
+    assert_eq!(
+        "Empty attribute key. Value: alpha",
+        Error::empty_attribute_key("alpha").to_string()
```

Comment:
Every change to the error message will require updating the tests, which feels more like an obstacle during development than something helpful.

Reply:
Yes, this is what I want, while all those variants are already stabilized and accidental changes should be directly reported in tests.